### ISSUE TITLE
Do not provide file system completion for Camel uri attribute

### ISSUE THREAD

**Issue by apupier** [eclipse-lemminx/lemminx#1060]
the uri attribute in Camel namespace is for Camel URI, not file system uri.

(I think it is the xml language server which is providing the file system completion entries)

**Comment by apupier** [comment]
for instance, the file completion entries are irrelevant here:
![XML DSL support Endpoint options filtering Duplicate endpoint options are filtered out](https://user-images.githubusercontent.com/1105127/122527745-e800e800-d01b-11eb-8eb4-ee707b80019d.png)

**Comment by angelozerr** [comment]
@apupier I agree with you, it's not relevant, but we provide (for any long time) this completion kind for any attribute https://github.com/redhat-developer/vscode-xml/blob/master/docs/Features/XMLFeatures.md#attribute-value-completion

But I has created an issue with limitation about that. I have add a comment at https://github.com/eclipse/lemminx/issues/786#issuecomment-863892890 please coment it.

**Comment by angelozerr** [comment]
This issue should be fixed a long ago, time since this file completion is not hard coded and works with settings https://github.com/redhat-developer/vscode-xml/blob/main/docs/Features/XMLFilePathSupport.md
